gammapy/stats/variability.py

```diff
@@ -289,6 +289,7 @@ def TimmerKonig_lightcurve_simulator(
     power_spectrum,
     npoints,
     spacing,
+    leakage_protection=1.0,
```

Comment:
Why the default is 1, and not 1.5 or 1.3?

Reply:
I think the default behavior should be to simulate the normal lightcurve, both for time efficiency and for consistency. A useful size of the multiplier could be 10 or 100.